app/controllers/navigation/Navigator.scala

```diff
@@ -17,14 +17,22 @@
 package controllers.navigation
 
 import config.AppConfig
+import controllers.util.{FormAction, SaveAndContinue, SaveAndReturn, Unknown}
 import javax.inject.Inject
 import models.requests.{ExportsSessionKeys, JourneyRequest}
 import models.responses.FlashKeys
-import play.api.mvc.{Result, Results}
+import play.api.mvc.{AnyContent, Call, Result, Results}
 
 class Navigator @Inject()(appConfig: AppConfig) {
 
-  def goToDraftConfirmation()(implicit req: JourneyRequest[_]): Result = {
+  def continueTo(call: Call)(implicit req: JourneyRequest[AnyContent]): Result =
+    FormAction.bindFromRequest match {
+      case Some(SaveAndReturn)          => goToDraftConfirmation()
+      case Some(SaveAndContinue) | None | Some(Unknown) => Results.Redirect(call)
```

Comment:
If pages just have a standard `<button type="submit">...</button>` you want them to continue as if it was a `SaveAndContinue`. This would appear as a `Some(Unknown)`.

I added `None` as well just because it has to be handled somewhere

Reply:
It's impossible to have None here, if there is None we're putting Unknown. Can we add `getOrElse(Unknown)` in `FormAction`? We will avoid some problems, it doesn't make any sense to put `Option[FormAction]` if we have `Unknown`. It's huge change, so it's not something that you must do it in this PR. If everyone agree I can make this change in separate PR.

Reply:
IMO we should fix the wrong implementation not add something to cover wrong implementation.

Reply:
@PatrykRudnicki I agree that it should default always to Unknown. I think fixing  it will be much easier after Save and Resume is done though so I'd leave it for now

Reply:
yes, it's not part of the ticket, and when we will minimize usage of this it will be easier to fix ;)